### DISCUSSION TITLE
Add "items" view with a table of items that is sortable, filterable, and editable

### DISCUSSION THREAD

**Discussion by philcunliffe** [sysadminsmedia/homebox#743] [Ideas]
### What is the problem you are trying to solve with this feature?

The power of the spreadsheet or notion or similar is the ability to see, sort, filter, and edit items all in place.

### What is the solution you are proposing?

A single, infinite scroll table with columns that are sortable, filterable, and editable. Notion does this well, see attached image.
![image](https://github.com/sysadminsmedia/homebox/assets/1517531/78810335-fe86-4aa5-8b2c-d9e96a8f19d8)

In addition to what notion does multi-select with actions like batch move would be good to consider from the start to inform the table choice or construction.

### Contributions

- [X] I have searched through existing issues and feature requests to see if my idea has already been proposed.
- [ ] If this feature is accepted, I would be willing to help implement and maintain this feature.
- [ ] If this feature is accepted, I'm willing to sponsor the development of this feature.

**Comment by katosdev** [comment]
By rights, this shouldn't be too difficult to implement. Will add this to the project tracker.

**Comment by tankerkiller125** [comment]
Please note we're converting this issue into a discussion as per our new procedures/policies. You can read more about that in our announcement #712

**Comment by boyerak** [comment]
See also [876](https://github.com/sysadminsmedia/homebox/discussions/876)
